middleware/model/openai.js

```diff
@@ -15,7 +15,7 @@ class Model {
     embed = async function(string){
         try{
             const resp = await this.model.embeddings.create({
-                model:"text-embedding-ada-002",
+                model:process.env.OPENAIEMBEDDINGMODEL,
```

Comment:
Please modify this so it automatically defaults to "text-embedding-ada-002" if the embedding model is not set.

Reply:
I have just pushed the changes